content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -378,12 +384,33 @@ Run `GameLauncher.exe` from your project game release layout, which is located i
 
 - `C:\MyProject\install\bin\Windows\release\Monolithic` -- For monolithic projects.
 
-Now you can distribute your project to other Windows devices. To run the Game Launcher, you must install the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160).
+
+## Distribute your build
+
+To distribute your build to other Windows computers:
+
+1. Create a zip folder of the `<install>\bin\Windows\release\<build>` folder. 
+
+    The contents of the zip folder should look similar to this: 
+
+        Monolithic
+        |   DevTestProject.GameLauncher.exe
+        |   PhysXDevice64.dll
+        |   PhysXGpu_64.dll
+        |   
+        \---Cache
+            \---pc
+            engine.pak
+                
+2. Distribute the zip folder in your preferred method such as through a file storage service. 
```

Comment:
```suggestion
2. Distribute the zip folder by using your preferred method such as through a file storage service. 
```

Reply:
```suggestion
1. Distribute the `.zip` file using your preferred method, such as a file storage service. 
```
* Changed "2." to "1.", [per our ordered list formatting guidelines](https://www.o3de.org/docs/contributing/to-docs/style-guide/format/#ordered-lists).
* Same changes as above for "zip folder".
* For a more natural flow, I recommend setting aside the example phrase ("such as...") using a comma.
* To sound slightly less formal, I think that we can drop the words "by" and "through".